torchvision/io/video.py

```diff
@@ -95,7 +95,7 @@ def _read_from_stream(container, start_offset, end_offset, stream, stream_name):
         # TODO check if stream needs to always be the video stream here or not
         container.seek(seek_offset, any_frame=False, backward=True, stream=stream)
     except av.AVError:
-        print("Corrupted file?", container.name)
+        # print("Corrupted file?", container.name)
```

Comment:
We should probably have some sort of "verbose" flag for this?

Reply:
yes, but we currently do not have any type of logging in torchvision. Printing to screen is definitely not something we want to do. And adding a logging option will take some more work and thought and I'd rather not rush things.

@soumith thoughts?

Reply:
I'm adding a TODO and removing the print for now